src/components/header/renderer.tsx

```diff
@@ -419,16 +415,15 @@ class Header extends React.PureComponent<Props, State> {
                 </span>,
                 portal(
                   <div className="modal-background" onClick={closePortal}>
-                    <div className="modal modal-top" onClick={e => e.stopPropagation()}>
-                      <div className="modal-header">
+                    <div className="modal" onClick={e => e.stopPropagation()}>
+                      <div>
                         <button className="close-button" onClick={closePortal}>
                           <X />
                         </button>
                       </div>
                       <div className="modal-body">
```

Comment:
Can we get rid of the modal body div?

Reply:
@domoritz I tried but it is messing up with the examples modal. Can you try for yourself once again?

Reply:
Yep, doesn't work.